docs/metrics-howto.md

```diff
@@ -7,7 +7,13 @@
 
 1.  Enable Synapse metrics:
 
-    There are two methods of enabling metrics in Synapse.
+    In `homeserver.yaml`, make sure `enable_metrics` is
```

Comment:
very aside to this PR, but I wonder why we even need/want this flag in the first place?
Shouldn't it be enough to have a metrics listener? I can't see why you'd want to change them independently...